api.planx.uk/modules/admin/session/csv.test.ts

```diff
@@ -23,7 +23,9 @@ jest.mock("@opensystemslab/planx-core", () => {
 });
 
 describe("CSV data admin endpoint", () => {
-  afterEach(() => jest.clearAllMocks());
+  afterEach(() => {
```

Comment:
hooks like `afterEach` have to return `null` or `undefined`, so have remove implicit return, as per [docs](https://vitest.dev/guide/migration.html#hooks)